bundles/pom.xml

```diff
@@ -30,4 +30,52 @@
     <module>org.openhab.ui.paperui</module>
   </modules>
 
+  <dependencies>
```

Comment:
Should we maybe just use a dependencyManagment section not to repeat the version number each time, afaik that would be the best practice?